src/platforms/rcore_desktop_rgfw.c

```diff
@@ -689,9 +691,8 @@ void EnableCursor(void)
 void DisableCursor(void)
 {
     RGFW_disableCursor = true;
-    
-    // Set cursor position in the middle
-    SetMousePosition(CORE.Window.screen.width/2, CORE.Window.screen.height/2);
+
+    RGFW_window_mouseHold(platform.window, RGFW_AREA(CORE.Window.screen.width / 2, CORE.Window.screen.height / 2));

```

Comment:
Not following raylib coding conventions, raylib does not use spaces between `*` and `/` operators